src/client/components/builder/EmbedModal.tsx

```diff
@@ -49,7 +51,17 @@ const EmbedField: FC<EmbedFieldProps> = ({ name, value, children }) => {
         <InputRightElement
           cursor="pointer"
           onClick={onClick}
-          children={<BiCopy />}
+          children={
+            <ReactIconTooltip
+              label="Copy"
+              placement="right"
+              openDelay={800}
+              gutter={16}
+              hasArrow
+            >
+              <BiCopy />
+            </ReactIconTooltip>
+          }
```

Comment:
```suggestion
          children={
            <Tooltip
              label="Copy"
              placement="right"
              openDelay={800}
              gutter={16}
              hasArrow
            >
              <span><BiCopy /></span>
            </Tooltip>
          }
```

Would this work? If so, we can do without the custom `ReactIconTooltip` component and only use the `Tooltip` Chakra component.